slither/printers/summary/when_not_paused.py

```diff
@@ -0,0 +1,72 @@
+"""
+    Module printing summary of the contract
+"""
+
+from typing import List, Set
+from slither.printers.abstract_printer import AbstractPrinter
+from slither.core.declarations.function import SolidityFunction
+from slither.utils.myprettytable import MyPrettyTable
+from slither.utils.function import get_function_id
+from slither.core.declarations import Contract, Function
+
+
+class PrinterWhenNotPaused(AbstractPrinter):
+
+    ARGUMENT = "when-not-paused"
+    HELP = "Print entry points that are not can not reach the modifier whenNotPaused"
+
+    WIKI = "https://github.com/trailofbits/slither/wiki/Printer-documentation#when-not-paused"
+
+    @staticmethod
+    def _list_functions_not_reaching_modifier(
+        contract: Contract, modifier_name: str = "whenNotPaused"
+    ) -> List[Function]:
+        results: Set[Function] = set()
+        for entry_point in contract.functions_entry_points:
+            if entry_point.visibility not in ["public", "external"]:
```

Comment:
Because this is iterating over `functions_entry_points`, there is no need to check for the visibility 

https://github.com/crytic/slither/blob/8344524cd35a1edf37c1cc791eb08c8ffff645a8/slither/core/declarations/contract.py#L487